squad-create-skipfile-reproducers

```diff
@@ -0,0 +1,251 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+# vim: set ts=4
+#
+# Copyright 2023-present Linaro Limited
+#
+# SPDX-License-Identifier: MIT
+
+import argparse
+import logging
+import os
+import pathlib
+import sys
+import time
+
+import yaml
+from squad_client.core.api import SquadApi
+
+import squadutilslib
+
+squad_host_url = "https://qa-reports.linaro.org/"
+SquadApi.configure(cache=3600, url=os.getenv("SQUAD_HOST", squad_host_url))
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+linux_stable_rc_branches = {
+    "linux-4.14.y": "linux-stable-rc",
+    "linux-4.19.y": "linux-stable-rc",
+    "linux-5.4.y": "linux-stable-rc",
+    "linux-5.10.y": "linux-stable-rc",
+    "linux-5.15.y": "linux-stable-rc",
+    "linux-6.1.y": "linux-stable-rc",
+    "linux-6.2.y": "linux-stable-rc",
+}
+
+linux_mainline_next_branches = {
+    "linux-mainline": "master",
+    "linux-next": "master",
+}
```

Comment:
As we spoke about we should probably go the other way around, look in SQUAD after projects that have been updated in LKFT's group the last 3 weeks. pull them and don't hardcode branches/projects like this.

Reply:
Noted - will experiment with this and then perform the update as discussed. Should be much easier to maintain the code this way.